curriculum/challenges/english/25-front-end-development/quiz-javascript-math/66edc3ab8c6413c344f401bf.md

```diff
@@ -17,439 +17,452 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+What is the result of this code:
+const num1 = Number('888');
+const num2 = new Number('777');
+
+console.log(num1 == 888); //statement 1
+console.log(num2 == 777); // statement 2
```

Comment:
````suggestion
What is the result of this code:

```js
const num1 = Number('888');
const num2 = new Number('777');

console.log(num1 == 888); //statement 1
console.log(num2 == 777); // statement 2
```
````

code needs to be in code fences. If you have other code snippets please wrap them in backticks too

Reply:
Thanks. I have made the changes